src/user-utilities.js

```diff
@@ -50,7 +50,21 @@ function createUser(firstName, lastName, email, phoneNumber) {
 
 }
 
+function   manageUsers(user, a) {
+  if ( a== "delete"){
```

Comment:
Extra spacing and === for strict comparison

Reply:
Strict comparison would be more apt here(===) rather than a loose comparison(==)

Also spacing after a for consistent spacing

"a" being called on may be confusing. We should use a more descriptive term.